server/src/controllers/score.controller.js

```diff
@@ -1,21 +1,33 @@
 import constants from '../utils/contants';
 import {
   queryScores,
+  queryScoresCount,
+  queryRank,
   queryUserScore,
   insertScore,
 } from '../utils/db_score_query';
 
-const {
-  HTTP_SUCCESS_OK,
-  HTTP_INTERNAL_SERVER_ERROR,
-  HTTP_CREATED,
-} = constants;
+const { HTTP_SUCCESS_OK, HTTP_INTERNAL_SERVER_ERROR, HTTP_CREATED } = constants;
 
 const getScores = async (req, res) => {
-  const { range } = req.params;
+  const { offset, limit } = req.query;
   try {
-    const scores = await queryScores(parseInt(range, 10));
-    res.status(HTTP_SUCCESS_OK).json(scores);
+    const scores = queryScores(parseInt(offset, 10), parseInt(limit, 10));
```

Comment:
```suggestion
    const scores = queryScores({ a: parseInt(offset, 10), b: parseInt(limit, 10) });
```
Try to use ^pattern for passing arguments. 
Pros: you will know what are the arguments name from the calling itself and also you don't have to remember the function position also

Reply:
Thanks for this excellent suggestion :)